google/cloud/testing_util/opentelemetry_matchers.h

```diff
@@ -150,6 +150,13 @@ MATCHER_P(SpanWithParent, span,
   return actual == span->GetContext().span_id();
 }
 
+MATCHER(SpanIsRoot, " is root span") {
```

Comment:
Nit:  The `MATCHER()` description doesn't need a leading space.

Reply:
Done